apps/web/public/manifest.json

```diff
@@ -6,16 +6,6 @@
       "src": "favicon.ico",
```

Comment:
I feel like we should rather remove this file completely or update it with the appropriate information. WDYT? @novuhq/novu-devs

Reply:
Yeah, @MaximFedarau update the `short_name` and `name` to:
```
  "short_name": "Novu Web",
  "name": "Novu Web application",``
```
for example

Reply:
Thanks, @LetItRock and @p-fernandez. Done.